homeworks/olga.tsarenko_olga-tsarenko/homework_2/homework_2.js

```diff
@@ -0,0 +1,22 @@
+let height = 4;
```

Comment:
1. variables block, space, and height have never reassigned, please use const
2. empty lines usually used for splitting logical blocks of code, but you do it for every line, please group them by logic.